ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -355,7 +355,8 @@ class AdminWorkspaceImpl extends React.Component<UrlParamsProps, State> {
           <h3>Basic Information</h3>
```

Comment:
I can not leave comment on the line that is not touched. Question for line 241 `getCloudStorageTraffic` above:
Two options 1: FE still passes namespace, and backend lookup workspace then convert to googleProject before calling google 
2:  Add a new endpoint with {googleProject} path, to replace the existing one:
https://github.com/all-of-us/workbench/blob/master/api/src/main/resources/workbench-api.yaml#L1350

Reply:
another similar question for line 320.
I will bring both questions in slack discussion tomorrow